src/components/card/Card.js

```diff
@@ -17,33 +19,106 @@ function tellHouseColor(hogwartsHouse) {
 
 function Card(props) {
   const houseColor = tellHouseColor(props.hogwartsHouse);
+  // const showDetails = true;
+  const [showDetails, setShowDetails] = useState(false);
+  const [emoji, setEmoji] = useState("");
 
   return (
     <section className="section-card">
       <div className="section-header">
         <div>
           <img className="section-img" src={props.imageURL} alt="" />
         </div>
-        <h2 className="section-header-title">{props.characterName}</h2>
+        <h2 className="section-header-title">
+          {emoji}
+          {props.characterName}
+        </h2>
         <div className={houseColor}>{props.hogwartsHouse}</div>
-        <button className="expand-more-button">
-          <img className="expand-more-img" src={expandMore} alt="" />
+        <button
+          className={!showDetails ? "expand-more-button" : "expand-less-button"}
+          onClick={() => {
+            setShowDetails(!showDetails);
+          }}
+        >
+          <img
+            className="expand-more-img"
+            src={!showDetails ? expandMore : expandLess}
+            alt=""
```

Comment:
Here you actually need an `alt` as this image is essential for the user to understand that you can get more information when clicking it.

Reply:
Done!